content/blog/2023/11/kubewarden-1.9-release.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Kubewarden 1.9.0 release
+authors:
+  - Flavio Castelli
+date: 2023-11-03
+---
+
+Not even a month after the 1.8.0 release, today we are happy to announce Kubewarden 1.9.0! 🎉🥳
+
+This release includes two major features that have been requested by our community.
+
+## Making Rego policies context-aware
+
+[Context-aware policies](https://docs.kubewarden.io/explanations/context-aware-policies) have been introduced
+with Kubewarden 1.6.0. These policies can obtain information about other Kubernetes resources at evaluation time.
+This allows them to make decisions based not only on the information provided by the `AdmissionReview` object they
+receive.
+
+Initially, this feature was available only to policies written using traditional programming languages. However, starting
+from the 1.9.0 release, the policies written using Rego can use cluster information.
+
+Kubewarden can run policies written in Rego that target both the
+[Gatekeeper](https://open-policy-agent.github.io/gatekeeper/website/) and the
+[`kube-mgmt`](https://github.com/open-policy-agent/kube-mgmt)
+platforms. In both cases, the Kubernetes information is shared with the policy using the same format used by these
+platforms. That means there's no need to rewrite your Gatekeeper/kube-mgmt policies. 😎
```

Comment:
:D.

This reminds me we can update the Gatekeeper comparison.

Reply:
You're right! I'm on it

Reply:
I forgot I've already done it :)

https://docs.kubewarden.io/next/explanations/opa-comparison